client/marketing/components/recommended-extensions/item.js

```diff
@@ -25,12 +26,17 @@ const RecommendedExtensionsItem = ( {
 	const classNameBase = 'woocommerce-marketing-recommended-extensions-item';
 	const connectURL = getInAppPurchaseUrl( url );
 
+	// Temporary fix to account for different styles between marketing & coupons
+	if ( category === 'coupons' && product === 'automatewoo' ) {
+		product = `automatewoo-alt`;
+	}
+
 	return (
 		<a href={ connectURL }
 			className={ classNameBase }
 			onClick={ onProductClick }
 		>
-			<ProductIcon src={ icon } />
+			<ProductIcon product={ product } />
```

Comment:
I'm seeing an error here that says "Element ProductIcon doesn't have required attribute src"

Reply:
Thanks, I'm not seeing the error in my browser but can know what the issue is